show_client/lldp_neighbors_cli.go

```diff
@@ -263,6 +265,18 @@ func getLLDPNeighbors(args sdc.CmdArgs, options sdc.OptionMap) ([]byte, error) {
 	// parse neighbors summary from full lldp data
 	neighbors := extractAllNeighborsEntry(data)
 
+	// If interface name is provided, filter to return only that interface
+	if ifaceName != "" {
+		if entry, ok := neighbors[ifaceName]; ok {
+			singleNeighbor := make(map[string]lldpNeighborsEntry)
```

Comment:
nit: you can simplify with just

`neighbors = map[string]lldpNeighborsEntry{ifaceName: entry}`